classes/PodsUI.php

```diff
@@ -2977,12 +2984,18 @@ public function manage( $reorder = false ) {
 							foreach ( $this->filters as $filter ) {
 								if ( isset( $this->pod->fields[ $filter ] ) ) {
 									$filter_field = $this->pod->fields[ $filter ];
+
+									if ( isset( $this->fields['manage'][ $filter ] ) && is_array( $this->fields['manage'][ $filter ] ) ) {
+										$filter_field = array_merge( $filter_field, $this->fields['manage'][ $filter ] );
+									}
 								} elseif ( isset( $this->fields['manage'][ $filter ] ) ) {
 									$filter_field = $this->fields['manage'][ $filter ];
 								} else {
 									continue;
 								}
-
+								?>
+								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-<?php esc_attr_e( $filter ); ?>">
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$filter".

<sub>hash: 6b439d2c7e6bfb448a038c5ee1457f10</sub>

Reply:
```suggestion
								<span class="pods-form-ui-filter-container pods-form-ui-filter-container-<?php echo esc_attr( $filter ); ?>">
```